internal/tools/update-go-azure-sdk/main.go

```diff
@@ -133,10 +133,14 @@ func run(ctx context.Context, input config) error {
 	}
 
 	// 3. Update the version of `hashicorp/go-azure-sdk` used in `terraform-provider-azurerm`
-	logger.Info("Updating `hashicorp/go-azure-sdk`..")
-	if err := updateVersionOfGoAzureSDK(ctx, input.azurermRepoPath, input.newSdkVersion); err != nil {
+	logger.Info("Updating the `hashicorp/go-azure-sdk` base layer..")
```

Comment:
looks like `determineChangesBetweenVersionsOfGoAzureSDK` also needs updating to check the changes for each of the Go modules